docs/databases/swift-kuery.html

```diff
@@ -252,237 +229,150 @@ <h2 class="heading-2"><span class="blue-text">Step 5 (Optional):</span> Test sav
+      <p>However the best way to test this would be to actually query the database in Swift rather than using the terminal.</p>
       <div class="underline"></div>
       <h2 class="heading-2"><span class="blue-text">Step 6:</span> Retrieve data from the database</h2>
-      <p>We need to modify our GET route to retrieve and respond with the data:</p>
-      <pre><code class="language-swift">app.router.get("/kuery") { request, response, next in
-    getTitles() { resp in
-      response.send(resp)
-      next()
+      <p>Inside our `selectHandler`, we need to get a connection from the connection pool:</p>
+      <pre><code class="language-swift">App.pool.getConnection() { connection, error in
+    guard let connection = connection else {
+        Log.error("Error connecting: \(error?.localizedDescription ?? "Unknown Error")")
+        return completion(nil, .internalServerError)
     }
-  }</code></pre>
-      <p>What we're doing here is calling an asynchronous `getTitle`.</p>
-      <p>As this call is asynchronous we only call the `response.send(resp)` when `getTitle` has finished executing.</p>
-      <p>Our project won't compile yet as we haven't defined the `getTitles` function, so we can do that now:</p>
-      <pre><code class="language-swift">func getTitles(_ callback: @escaping (String) -> Void ) -> Void {
-
-  }</code></pre>
-      <p>Once again our project will now compile but nothing will happen as we haven't added any logic to our function.</p>
-      <p>Just like we did in step 4 we need to get a connection from the connection pool:</p>
-      <pre><code class="language-swift">func getTitles(_ callback: @escaping (String) -> Void ) -> Void {
-    pool.getConnection() { connection, error in
-      guard let connection = connection else {
-        guard let error = error else {
-          Log.error("Error connecting: Unknown Error")
-          return
-        }
-        Log.error("Error connecting: \(error)")
-        return
-      }
-    }
-  }</code></pre>
-      <p>Now we can build our query that will query the database for every entry in the "Titles" database:</p>
-      <pre><code class="language-swift">func getTitles(_ callback: @escaping (String) -> Void ) -> Void {
-    pool.getConnection() { connection, error in
-      guard let connection = connection else {
-        guard let error = error else {
-          Log.error("Error connecting: Unknown Error")
-          return
-        }
-        Log.error("Error connecting: \(error)")
-        return
-      }
-      let selectQuery = Select(from: titles)
-    }
-  }</code></pre>
+    // Write query and execute it here
+}</code></pre>
+      <p>Now we can build our SELECT query that will query the database for every entry in the "BookTable":</p>
+      <pre><code class="language-swift">let selectQuery = Select(from: App.bookTable)</code></pre>
       <p>Like before we can now execute the query using the `execute` method in Connection:</p>
```

Comment:
The wording here is clunky... how about something like...
"We can now execute our query using `connection.execute`:"
it would be nice to link to the apidoc for connection.execute`.